src/main/java/org/apache/accumulo/access/AccessEvaluatorImpl.java

```diff
@@ -230,4 +226,8 @@ public AccessEvaluator build() {
   public static AuthorizationsBuilder builder() {
     return new BuilderImpl();
   }
+
+  private static boolean isQuoteOrSlash(byte b) {
```

Comment:
I think this implementation might perform better. For each invocation of the current method, the input argument `b` has to go through a conversion from `byte` to `char` as described [here](https://docs.oracle.com/javase/specs/jls/se11/html/jls-5.html#jls-5.1.4).

```
    private static final byte QUOTE = (byte) '"';
    private static final byte BACKSLASH = (byte) '\\';

    private static boolean isQuoteOrSlash(byte b) {
      return b == quote || b == backslash;
    }
```

Reply:
Do you mean in places like:
```
          escapedAuth[index++] = '\\'
```

If so, then I think `yes` we should as `escapedAuth` is a `byte[]`, not a `char[]`. It will save us time on the conversion.

Reply:
Take a look at the additions made in 096df65. Created a new class to house these constants and compare them.

Reply:
Comments below

Reply:
Addressed your comments in f89ecd0